src/backend/InvenTree/order/templates/order/order_base.html

```diff
@@ -75,10 +77,38 @@
         {% endif %}
     </ul>
 </div>
-{% if order.is_pending %}
+{% if order.is_pending and order.requires_approval %}
```

Comment:
This templating is a complex mess... not sure if there's a way around this though. Let's expedite the move to platform!